data/brands/shop/alcohol.json

```
@@ -61,6 +61,17 @@
         "shop": "alcohol"
       }
     },
+    {
+      "displayName": "Binny's Beverage Depot",
+      "id": "binnysbeveragedepot-2740a5",
+      "locationSet": {"include": ["us"]},
+      "tags": {
+        "brand": "Binny's Beverage Depot",
+        "brand:wikidata": "Q2632011",
```

Looks like [Q2632011](https://www.wikidata.org/wiki/Q2632011) just points to the generic wikidata for a "liquor store".  

What we really need is to lookup the Wikipedia/Wikidata for "Binny's Beverage Depot".  I did a quick Google search to find that it should be these:

```suggestion
        "brand:wikidata": "Q30687714",
        "brand:wikipedia": "en:Binny's Beverage Depot",
```